src/app/_components/SplitContentCard.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image'
+
+import { clsx } from 'clsx'
+
+import type { StaticImageProps } from '@/types/sharedProps/imageType'
+
+import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
+
+import { BasicCard } from '@/components/BasicCard'
+
+type SplitContentCardProps = {
+  index: number
+  image: StaticImageProps
+  as?: React.ElementType
+  children: React.ReactNode
+}
+
+export function SplitContentCard({
+  index,
+  image,
+  as: Tag = 'div',
+  children,
+}: SplitContentCardProps) {
+  const isImagePositionOdd = index % 2 === 1
+
+  return (
+    <Tag
+      className={clsx(
+        'flex flex-col gap-4 lg:flex-row lg:gap-6',
```

Comment:
In Figma, the gap between the image and the content (`BasicCard`) is always the same as the grid gap.

Here, the gap is set to `gap-4 lg:gap-6` while the grid gap, defined in `PageSection`, is `gap-6`. Hence, on screens smaller than `lg`, this results in wider gaps between cards than within the cards.

Does that make sense?

Not sure if it's a fix for the PR, just flagging it :)

Reply:
This seems like the intended behavior to me—to have more space between two `AlternatingImageCards` on smaller screens than between the elements within a single `AlternatingImageCard`. I believe adding even more space could be beneficial to make it clearer which image belongs to which card. I’m happy to align this with the design.